mne/io/pick.py

```diff
@@ -15,7 +15,7 @@
                      _check_option)
 
 
-def get_channel_types():
+def get_channel_type_constants():
```

Comment:
It’s public function we need to deprecate it properly. It’s used in mnelab

Reply:
oops. yep, I already forgot my own suggestion :)  Will fix.